pycode/memilio-epidata/memilio/epidata/getPopulationData.py

```diff
@@ -35,203 +38,197 @@
+
+    df_pop_export[dd.EngEng['population']
+                  ] = df_pop_export.iloc[:, 2:].sum(axis=1)
+
+    if df_pop_export[dd.EngEng['population']][df_pop_export.ID_County == '16056'].values == 0:
+        df_pop_export = geoger.merge_df_counties_all(
+            df_pop_export, sorting=[dd.EngEng["idCounty"]],
+            columns=dd.EngEng["idCounty"])
 
-    directory = os.path.join(out_folder, 'Germany/')
     gd.check_dir(directory)
 
-    filename_counties = 'county_table'
-    filename_zensus = 'zensus'
-    filename_reg_key = 'reg_key'
-
-    url_zensus = 'https://opendata.arcgis.com/datasets/abad92e8eead46a4b0d252ee9438eb53_1.csv'
-    url_reg_key = 'https://www.zensus2011.de/SharedDocs/Downloads/DE/Pressemitteilung/DemografischeGrunddaten/' \
-        '1A_EinwohnerzahlGeschlecht.xls?__blob=publicationFile&v=5'
-    path_zensus = os.path.join(directory, filename_zensus + ".json")
-    path_reg_key = os.path.join(directory, filename_zensus + ".json")
-    zensus = gd.get_file(
-        path_zensus, url_zensus, read_data, param_dict={}, interactive=True)
-    reg_key = gd.get_file(path_reg_key, url_reg_key, read_data, param_dict={
-        "engine": None, "sheet_name": 'Tabelle_1A', "header": 12}, interactive=True)
-    counties = geoger.get_official_county_table()
+    if len(df_pop_export) == 401:
+        filename = 'county_current_population_dim401'
+        gd.write_dataframe(df_pop_export, directory, filename, file_format)
 
-    if not read_data:
-        if not no_raw:
-            if not counties.empty:
-                gd.write_dataframe(counties, directory,
-                                   filename_counties, file_format)
-            if not zensus.empty:
-                gd.write_dataframe(zensus, directory,
-                                   filename_zensus, file_format)
-            if not reg_key.empty:
-                gd.write_dataframe(reg_key, directory,
-                                   filename_reg_key, file_format)
+    if len(df_pop_export) == 400 or merge_eisenach == True:
+        filename = 'county_current_population'
+
+        # Merge Eisenach and Wartburgkreis
+        df_pop_export = geoger.merge_df_counties_all(
+            df_pop_export, sorting=[dd.EngEng["idCounty"]],
+            columns=dd.EngEng["idCounty"])
+
+        gd.write_dataframe(df_pop_export, directory, filename, file_format)
+
+    return df_pop_export
+
+
+def assign_population_data(df_pop_raw, counties, age_cols, idCounty_idx):
+    '''Creates new dataframe and copies values of counties from old dataframe'''
+
+    new_cols = {dd.EngEng['idCounty']: counties[:, 1],
+                dd.EngEng['county']: counties[:, 0]}
+
+    # number of age_cols
+    nage = len(age_cols)
+
+    # add age_cols with zero initilization to new_cols
+    new_cols.update({age: 0 for age in age_cols})
 
-    return counties, zensus, reg_key
+    df_pop = pd.DataFrame(new_cols)
+
+    empty_data = ['.', '-']
+    for i in idCounty_idx:
+
```

Comment:
```suggestion
county_id = df_pop_raw.loc[i, dd.EngEng['idCounty']]
```
Instead of repeating calls.